spec/lib/rmt/cli/products_spec.rb

```diff
@@ -222,19 +252,38 @@
         end
 
         it 'has more repositories than the base product' do
-          expect(repo_count).to be > product.repositories.where(enabled: true).count
+          expect(repos.count).to be > product.repositories.where(enabled: true).count
+        end
+      end
+    end
+
+    context 'by wrong product string' do
+      let(:target) { 'badproductstring' }
+      let(:argv) { ['enable', target] }
+      let(:expected_stderr) { "Product(s) #{target} could not be found and were not enabled.\n" }
+      let(:expected_output) { "No product found for target '#{target}'.\n" }
+
+      before do
+        expect(described_class).to receive(:exit)
```

Comment:
Not happy with `expect` blocks in `before` blocks. In this specific case we can use `allow(...)`.